### DISCUSSION TITLE
Multi application versions

### DISCUSSION THREAD

**Discussion by OskarLebuda** [semantic-release/semantic-release#1806] [Configuration] [answered]
Hey. Maybe there were some discussion about it. BUT I have a question about multiapp versioning. 

We have 1 repo with 1 `package.json` but we need to version it separently.

Example:
```
Apps/
    |_ App1
    |_ App2
Core/
Engine/
.releaserc
package.json
```

We need to create tags for each app with suffix of this app. Each will have separated versioning.
example:
`v1.39.0-app1` and `v1.0.3-app2`
If we add new app we have to create new version of new app `v1.0.0-app3`

have you any idea how to do this automaticly?

EDIT:

here is example of git repository https://bl.ocks.org/OskarLebuda/raw/4369692757f3a5538643c65f80cb8318/

**Comment by gr2m** [comment, marked as answer]
there is https://github.com/atlassian/lerna-semantic-release. `semantic-release` itself does currently not support any concept of a mono repo

**Reply by OskarLebuda** [reply]
Yea, but out project doesn't use Lerna. We use webpack to configure current app

**Comment by bigbrozer** [comment]
Hi @OskarLebuda, I had a similar situation for a mono-repo project and find out [pmowrer/semantic-release-monorepo](https://github.com/pmowrer/semantic-release-monorepo). I did not test it yet, but this is a good candidate ;-) Additionally, it has lerna support (not mandatory, this is an extra option).

**Reply by travi** [reply]
this is the one that i use in a couple of monorepos that i manage and have been pleased with it
